src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -372,4 +365,20 @@ private void checkSampleInBamHeader(final String sample) {
     private String decodeSampleNameIfNecessary(final String name) {
         return samplesList.asListOfSamples().contains(name) ? name : IOUtils.urlDecode(name);
     }
+
+    public String getVersion() {
```

Comment:
Did you add these intentionally?

Reply:
I did, would it be better to make the fields not private?

Reply:
I was thinking move `getMutect2VCFHeaderLines` from `Mutect2` to `Mutect2Engine` and then these fields could remain private without needing to add the getters.